setup.py

```diff
@@ -157,7 +157,7 @@ def run(self):
     if 'dev' in '0.12.0.dev99999999999999':
         current_version = datetime.datetime.now().strftime('%Y%m%d%H%M%S')
         call(['python3', '.misc/adjust_version_number.py', 'coalib/VERSION',
-              '-b {}'.format(current_version)])
+              f'-b {current_version}'])
```

Comment:
either revert changes in this file, to get green Travis CI build or make changes in coala Moban repository and after there, changes are merged re-run Travis CI build to get it green.

Either of the solutions can be used.

Reply:
Hey @jkklapp looks like you reverted change only the line i added review. There were more changes in the same file. Please look after them as well

Reply:
@KVGarg I did what you asked and the jobs keep failing

Reply:
No problem! The build now failed is not because of your changes now. It has broken now due to package not found error - "setuptools-git" 

Issue is created for the same

https://github.com/coala/coala/issues/6165

As soon it is fixed, we can get your branch rebase and get your PR green.